extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -278,6 +295,16 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * Receives any response that may be sent from the cross-domain iframe
+   * @param {String} msg The response message from the iframe
+   * was specified in the amp-analytics config
+   */
+  processCrossDomainIframeResponse_(msg) {
+    // Used in ../../../src/anchor-click-interceptor.js
+    this.win.document.mostRecentCrossDomainIframeResponse = msg;
```

Comment:
^^ Reviewers: extra attention requested on this line. Is there a better place to store this? The logic is that, when there are multiple ads with amp-analytics tags on the page, then if the user clicks on ad A, we want to use the most recent response from ad A. But if they click on ad B, we want to use the most recent response from ad B. So I've stored the response as an attribute on the ad's document (not the containg AMP publisher page). Then anchorClickInterceptor pulls it from the event target's document.